Dockerfile

```
@@ -0,0 +1,12 @@
+FROM golang:1.16 as build
+WORKDIR /app
+COPY . .
+RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build github.com/planetscale/cli/cmd/pscale
+
+FROM alpine:latest  
+RUN apk --no-cache add ca-certificates
+RUN apk add mysql-client
```

```suggestion
RUN apk --no-cache add ca-certificates mysql-client
```

should we do this all in one `RUN` command?

---

Yeap, makes sense 👍🏼